components/collections.js

```diff
@@ -51,7 +52,7 @@ export default function Collections({ collections = null }) {
           return (
             <Link
               key={collection}
-              href={`/site-search/?term=${collection}`}
+              href={`/search/?type=${itemType}&collections=${collection}&status=released`}
```

Comment:
Do we need status here since the user with certain access may also want to see data that is not released?